<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test one"
           tests="4"
           failures="4"
           errors="0"
           skipped="0"
           time="8.085">
  <properties />
            <testcase name="SignupPopup" time="2.011" classname="testNG.KatkadaWebPage">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  testNG.KatkadaWebPage.SignupPopup(KatkadaWebPage.java:34)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:567)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="Signin" time="2.060" classname="testNG.KatkadaWebPage">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
  org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
  com.sun.proxy.$Proxy12.click(Unknown Source)
  testNG.KatkadaWebPage.Signin(KatkadaWebPage.java:24)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:567)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="ForgetpasswordPopup" time="2.003" classname="testNG.KatkadaWebPage">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  testNG.KatkadaWebPage.ForgetpasswordPopup(KatkadaWebPage.java:63)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:567)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="SignInButton" time="2.011" classname="testNG.KatkadaWebPage">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  testNG.KatkadaWebPage.SignInButton(KatkadaWebPage.java:88)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:567)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
